sdks/python/apache_beam/testing/test_utils.py

```diff
@@ -32,6 +35,46 @@
 DEFAULT_HASHING_ALG = 'sha1'
 
 
+class TempDir(object):
```

Comment:
<!--new_thread; commit:5d08ca3beac2fd751bbfdcf83d47e57ce1411bc0; resolved:0-->
Thanks for adding this. Would you mind adding a test to test_utils_test.py as well ?

Reply:
Done.